content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning.md

```diff
@@ -1053,13 +1053,14 @@ The following is an example procedure to upgrade from `v1beta1` to `v1`.
 3.  Remove `v1beta1` from the CustomResourceDefinition `status.storedVersions` field.
 
 {{< note >}}
-The `kubectl` tool currently cannot be used to edit or patch the `status` subresource on a CRD: see the [Kubectl Subresource Support KEP](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cli/2590-kubectl-subresource) for more details.
+In Kubernetes v1.24, an alpha flag `--subresource` is added to get, patch, edit and replace 
+with kubectl commands to fetch and update `status` and `scale` subresources. Due to this, 
+from Kubernetes v1.24 the `kubectl` tool can now be used to edit or patch the `status`
+subresource on a CRD. See [How to patch deployment by using subresource flag](/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch/#Use-subresource-flag-with-kubectl-patch-command-to-update-the-deployment).
```

Comment:
I'm not happy with this text. It sounds like too much as if it was written before Kubernetes v1.24 was released.
Instead, please describe how the latest Kubernetes release works, right now.

Reply:
addressed